yarn-project/simulator/src/client/client_execution_context.ts

```diff
@@ -65,7 +65,7 @@ export class ClientExecutionContext extends ViewDataOracle {
     protected readonly contractAddress: AztecAddress,
     private readonly argsHash: Fr,
     private readonly txContext: TxContext,
-    private readonly callContext: CallContext,
+    public readonly callContext: CallContext,
```

Comment:
Why does it become public?

Reply:
Whoops, 😅 Nice catch, forgot from debugging.